src/strategies/uxlink-staking-vote-v1/index.ts

```diff
@@ -0,0 +1,41 @@
+import { formatUnits } from '@ethersproject/units';
+import { getAddress } from '@ethersproject/address';
+
+import { Multicaller } from '../../utils';
+import { BigNumberish } from '@ethersproject/bignumber';
+
+export const author = 'lessthanno';
+export const version = '0.1.0';
+
+const abi = [
+  'function getUserStakeInfo(address user) external view returns (uint256, uint256, uint256, uint256)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  const multi = new Multicaller(network, provider, abi, { blockTag });
+
+  addresses.forEach((address) =>
+    multi.call(address, options.address, 'getUserStakeInfo', [address])
+  );
+
+  const result: Record<string, BigNumberish> = await multi.execute();
+
+  return Object.fromEntries(
+    Object.entries(result).map(([address, userStakeInfo]) => {
+      const checksum = getAddress(address);
+      return [
+        checksum,
+        parseFloat(formatUnits(userStakeInfo[1], options.decimals))
+      ];
+    })
+  );
```

Comment:
Hey @lessthanno you can use the `contract-call` strategy in your space instead of a separate strategy. could you try it?

Reply:
I'll try.

Reply:
It should work, i have added your parameters here https://snapshot.org/#/playground/contract-call?query=eyJwYXJhbXMiOnsiYWRkcmVzcyI6IjB4QmQzMzQ4MzhGMEIzODE3MThmNzRBNDA0MTRiNTdFQ0RDYkFGREFjRSIsInN5bWJvbCI6IlVYTElOSyIsImRlY2ltYWxzIjoxOCwib3V0cHV0IjoiMSIsIm1ldGhvZEFCSSI6eyJpbnB1dHMiOlt7ImludGVybmFsVHlwZSI6ImFkZHJlc3MiLCJuYW1lIjoidXNlciIsInR5cGUiOiJhZGRyZXNzIn1dLCJuYW1lIjoiZ2V0VXNlclN0YWtlSW5mbyIsIm91dHB1dHMiOlt7ImludGVybmFsVHlwZSI6InVpbnQyNTYiLCJuYW1lIjoiIiwidHlwZSI6InVpbnQyNTYifSx7ImludGVybmFsVHlwZSI6InVpbnQyNTYiLCJuYW1lIjoiIiwidHlwZSI6InVpbnQyNTYifSx7ImludGVybmFsVHlwZSI6InVpbnQyNTYiLCJuYW1lIjoiIiwidHlwZSI6InVpbnQyNTYifSx7ImludGVybmFsVHlwZSI6InVpbnQyNTYiLCJuYW1lIjoiIiwidHlwZSI6InVpbnQyNTYifV0sInN0YXRlTXV0YWJpbGl0eSI6InZpZXciLCJ0eXBlIjoiZnVuY3Rpb24ifX0sIm5ldHdvcmsiOiI0MjE2MSIsInNuYXBzaG90IjoiMjU2OTI0NDUxIiwiYWRkcmVzc2VzIjpbIjB4RTlhNjM3MjcxRTlBRGJFMzU1Q2RBOWM1YTU1ODUyQzA1ODc5NDJEZiJdfQ..

you have to setup your space accordingly

Reply:
Yes, I can run it too, thanks so much!